Multisig/Info.plist

```diff
@@ -21,7 +21,7 @@
 	<key>CFBundlePackageType</key>
 	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
 	<key>CFBundleShortVersionString</key>
-	<string>3.6.0</string>
+	<string>3.7.0</string>
```

Comment:
@MouazAlzahabi it looks like you pulled from `main` branch, can you undo this version change please?